internal/databases/mysql/mysql_test.go

```diff
@@ -40,13 +41,15 @@ func TestGetBinlogConfigNoError(t *testing.T) {
 
 func TestBinlogShouldBeFetched(t *testing.T) {
```

Comment:
I guess there should be "Binlogs" instead of "binlog"

Reply:
And it would be better to extract function BinlogShouldBeFetched for one binlog. And try to give test  name that shows expected result. https://dzone.com/articles/7-popular-unit-test-naming